src/main/scala/com/hortonworks/spark/sql/hive/llap/HS2JDBCWrapper.scala

```diff
@@ -20,11 +20,15 @@ package com.hortonworks.spark.sql.hive.llap
 import java.net.URI
 import java.sql.{Connection, DatabaseMetaData, Driver, DriverManager, ResultSet, ResultSetMetaData, SQLException}
 
```

Comment:
@merlintang  We need a corresponding PR for master and 2.1. If this part of the code is not different across branches then this should land in master. Then 2.1 and then 1.6 branches.